uprotocol/transport/builder/uattributesbuilder.py

```diff
@@ -134,6 +140,16 @@ def withToken(self, token: str):
         """
         self.token = token
         return self
+    
+    def withSource(self, source: UUri):
```

Comment:
If we add source to the constructor then this method is not needed.

Reply:
Removed it